jhucourse/woj/backend/game.py

```diff
@@ -0,0 +1,134 @@
+'import json'
+import numpy
+import player
+'import QuestionBank'
+
+class game:
+    def __init__(self):
+        self.player_list = []
+        self.current_player = 0
+        self.spin_counter = 50
+        self.current_round = 1
+        self.turnPoints = 0
+        self.question_bank
```

Comment:
self.question_bank should be set to None

Reply:
At what point will it be initialized? Or is that not something for me to worry about in this class?